Automattic/vip-cli
# [dev-env] Introuces update to change existing environment

## Description

This command is usefull when want to tweak existing environment. Common scenarios are:
* Enable/disable phpmyadmin/statsd/xdebug
* Switch wp version


## Steps to Test

1) Create environment
2) Run `vip dev-env update`
3) Make sure it does what you would expect it to do and correct default values are fetched from existing env
